src/Interpreters/TableJoin.cpp

```diff
@@ -135,7 +139,8 @@ void TableJoin::resetCollected()
 
 void TableJoin::addUsingKey(const ASTPtr & ast)
 {
-    addKey(ast->getColumnName(), renamedRightColumnName(ast->getAliasOrColumnName()), ast);
+    /// For USING key and right key AST is the same.
```

Comment:
Just because we specify key once: `SELECT ... FROM t1 JOIN t2 USING (key)` both left and right key defined by ASTIdentifier `key`. I added this example to comment either.

Reply:
Yep, agree.